claims_hosp/tests/test_backfill.py

```diff
@@ -0,0 +1,83 @@
+import os
+import glob
+from datetime import datetime
+
+# third party
+import pandas as pd
+import pytest
+
+# first party
+from delphi_claims_hosp.config import Config, GeoConstants
+from delphi_claims_hosp.backfill import store_backfill_file, merge_backfill_file
+
+CONFIG = Config()
+CONSTANTS = GeoConstants()
+PARAMS = {
+    "indicator": {
+        "input_file": "test_data/SYNEDI_AGG_INPATIENT_11062020_1451CDT.csv.gz",
+        "backfill_dir": "./backfill",
+        "drop_date": "2020-06-11",
+    }
+}
+DATA_FILEPATH = PARAMS["indicator"]["input_file"]
+DROP_DATE = pd.to_datetime(PARAMS["indicator"]["drop_date"])
+backfill_dir = PARAMS["indicator"]["backfill_dir"]
+
+class TestBackfill:
+
+    def test_store_backfill_file(self):
+        dropdate = datetime(2020, 1, 1) 
+        fn = "claims_hosp_as_of_20200101.parquet"
+        assert fn not in os.listdir(backfill_dir)
+       
+        # Store backfill file
+        store_backfill_file(DATA_FILEPATH, dropdate, backfill_dir)
+        assert fn in os.listdir(backfill_dir)
+        fn = "claims_hosp_as_of_20200101.parquet"
+        backfill_df = pd.read_parquet(backfill_dir + "/"+ fn, engine='pyarrow')
+        
+        selected_columns = ['time_value', 'fips', 'state_id',
+                        'num', 'den']
+        assert set(selected_columns) == set(backfill_df.columns)  
+        
+        os.remove(backfill_dir + "/" + fn)
+        assert fn not in os.listdir(backfill_dir)
+        
+    def test_merge_backfill_file(self):
+        
+        today = datetime.today()
+        
+        new_files = glob.glob(backfill_dir + "/claims_hosp*.parquet")
```

Comment:
These files are pretty large -- 3MB each. Do they need to be that large in order to test this behavior, or would smaller files work?

Reply:
We can use smaller test files instead.